physics/sfc_diff.f

```diff
@@ -78,6 +78,7 @@ subroutine sfc_diff_run (im,rvrdm1,eps,epsm1,grav,                &  !intent(in)
      &                      fh2_wat,   fh2_lnd,   fh2_ice,              &  !intent(inout)
      &                    ztmax_wat, ztmax_lnd, ztmax_ice,              &  !intent(inout)
      &                    zvfun,                                        &  !intent(out)
+     &                    use_lake_model,                               &  !intent(in)
```

Comment:
Why was `use_lake_model` added as an argument if it is not used?

Reply:
It was used at one point. We must have removed the code that used it.